tez-runtime-internals/src/main/java/org/apache/tez/runtime/task/TezChild.java

```diff
@@ -503,6 +506,32 @@ public static TezChild newTezChild(Configuration conf, String host, int port, St
         hadoopShim);
   }
 
+  private static void eagerInitFileSystemPaths(Configuration conf) {
+    Collection<String> eagerInitPaths = conf.getTrimmedStringCollection(
+        TezConfiguration.TASK_EAGER_INIT_FS_PATHS);
+    if (eagerInitFsPool == null && !eagerInitPaths.isEmpty()) {
+      eagerInitFsPool = Executors.newCachedThreadPool(new ThreadFactoryBuilder()
+          .setDaemon(true)
+          .setPriority(Thread.MAX_PRIORITY)
```

Comment:
in the code, I can see we tend not to set priority in thread pools...I guess we can remove this to simplify this code further

Reply:
ack.